### PR TITLE
Massimport for users

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
Now it's very fast (also with groups) using atomic transactions to the db.

### CODE REVIEW

File: openslides/users/static/js/users/site.js [hunk 1]
```diff
@@ -898,34 +897,46 @@ angular.module('OpenSlidesApp.users.site', [
...
-                User.create(user).then(
-                    function(success) {
-                        $scope.importcounter++;
-                    }
-                );
+            });
+            $http.post('/rest/users/user/mass_import/', {
+                users: users
+            }).then(function (success) {
+                console.log(success);
```
Comment (normanjaeckel) [inline openslides/users/static/js/users/site.js]: Please remove this line.

File: openslides/users/views.py [hunk 2]
```diff
@@ -114,6 +119,45 @@ def reset_password(self, request, pk=None):
...
+                groups = data['groups_id']
+                del data['groups_id']
+
+                db_user = User(**data)
+                db_user.save(skip_autoupdate=True)
+                db_user.groups.add(*groups)
+                created_users.append(db_user)
+                if 'importTrackId' in user:
+                    imported_track_ids.append(user['importTrackId'])
+            except ValidationError:
```
Comment (normanjaeckel) [inline openslides/users/views.py]: Please catch this exception only at the line that may raise it (serializer.is_valid(raise_exception=True) I think).

File: openslides/users/static/js/users/site.js (hunk 1 above)
Comment (FinnStutzenstein) [inline openslides/users/static/js/users/site.js]: Done.

File: openslides/users/views.py (hunk 2 above)
Comment (FinnStutzenstein) [inline openslides/users/views.py]: Done.

**Pull request merged**